runtime/queries/lua/highlights.scm

```diff
@@ -216,7 +243,5 @@
 
 (string) @string @spell
 
-(escape_sequence) @string.escape
-
```

Comment:
Interesting 🤔 This never got added in `nvim-treesitter`, but it was here. And it's supported in `tree-sitter-lua` since `v0.0.15`.

Reply:
Thanks; I added this here and meant to downstream it back (but forgot). I didn't mean to remove it here.